### PR TITLE
donationform, mainpageDoctor and mainpageDonor implementation

### PR DESCRIPTION
**Pull request by BocsaMirela**

### CODE REVIEW

File: src/main/java/ui_layer/controllers/MainController.java [hunk 1]
```diff
@@ -27,4 +27,19 @@ public String getAllUsers(Model model) {
     public String showLastDonationPage(Model model) {
         return "lastdonation";
     }
+	
+	@GetMapping("/DonationForm")
+    public String showDonationFormPage(Model model) {
+        return "DonationForm";
```
Comment (dariusgalanton) [inline src/main/java/ui_layer/controllers/MainController.java]: You could've pressed a CTRL+L for code formatting before commiting :) .
Please do that next time before opening a pull request.

File: src/main/resources/templates/js/donationform.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+$(document).ready(function(){
+    $(document).delegate("select", "change", function() {
+        //capture the option
+        $('.div_special').hide();
+        var $target = $("option:selected",$(this));
+        //$(".label_doctor").show();
```
Comment (dariusgalanton) [inline src/main/resources/templates/js/donationform.js]: Commented code should be removed.

File: src/main/resources/templates/js/mainPageDonator.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        showSlides(slideIndex = n);
+    }
+
+    function showSlides(n) {
+        var i;
+        var slides = document.getElementsByClassName("mySlides");
+        var dots = document.getElementsByClassName("dot");
+        if (n > slides.length) {slideIndex = 1}
+        if (n < 1) {slideIndex = slides.length}
+        for (i = 0; i < slides.length; i++) {
```
Comment (dariusgalanton) [inline src/main/resources/templates/js/mainPageDonator.js]: Empty line before a for and one after its last bracket improves readability.

File: src/main/resources/templates/js/mainPageDonator.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+            slides[i].style.display = "none";
+        }
+        for (i = 0; i < dots.length; i++) {
+            dots[i].className = dots[i].className.replace(" active", "");
+        }
```
Comment (dariusgalanton) [inline src/main/resources/templates/js/mainPageDonator.js]: Empty line before a for and one after its last bracket improves readability.

File: src/main/resources/templates/mainPageDoctor.html [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+    </div>
+    <div class="panel-body">
+            <div class="harvestdate">
+                <span>
+
+
+                </span> <br>
+                <div class="form-control-plaintext" id="nextDonation" ><br>
+                    Text sugestiv Text sugestiv Text sugestivv Text sss sugestiv Text sugestiv Text ssss sugestivv Text sugestiv
+                    Text sugestivv Text sss sugestiv Text sugestiv Text ssss sugestivv Text sugestiv</div>
```
Comment (dariusgalanton) [inline src/main/resources/templates/mainPageDoctor.html]: What's this? Is this text really needed?

File: src/main/resources/templates/mainPageDoctor.html [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+        showSlides(slideIndex = n);
+    }
+
+    function showSlides(n) {
+        var i;
+        var slides = document.getElementsByClassName("mySlides");
+        var dots = document.getElementsByClassName("dot");
+        if (n > slides.length) {slideIndex = 1}
+        if (n < 1) {slideIndex = slides.length}
+        for (i = 0; i < slides.length; i++) {
```
Comment (dariusgalanton) [inline src/main/resources/templates/mainPageDoctor.html]: Empty line before a for and one after its last bracket improves readability.

File: src/main/resources/templates/mainPageDoctor.html [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+            slides[i].style.display = "none";
+        }
+        for (i = 0; i < dots.length; i++) {
+            dots[i].className = dots[i].className.replace(" active", "");
```
Comment (dariusgalanton) [inline src/main/resources/templates/mainPageDoctor.html]: Empty line before a for and one after its last bracket improves readability.

File: src/main/resources/templates/mainPageDonator.html [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+        var i;
+        var slides = document.getElementsByClassName("mySlides");
+        var dots = document.getElementsByClassName("dot");
+        if (n > slides.length) {
+            slideIndex = 1
+        }
+        if (n < 1) {
+            slideIndex = slides.length
+        }
+        for (i = 0; i < slides.length; i++) {
```
Comment (dariusgalanton) [inline src/main/resources/templates/mainPageDonator.html]: Empty line before a for and one after its last bracket improves readability.

File: src/main/resources/templates/mainPageDonator.html [hunk 9]
```diff
@@ -0,0 +1,115 @@
...
+            slides[i].style.display = "none";
+        }
+        for (i = 0; i < dots.length; i++) {
+            dots[i].className = dots[i].className.replace(" active", "");
```
Comment (dariusgalanton) [inline src/main/resources/templates/mainPageDonator.html]: Empty line before a for and one after its last bracket improves readability.

**Comment by norberthcsorba** [comment]
screenshot 24

that doesn't look very good :)) but it will be future work

**Comment by norberthcsorba** [comment]
screenshot 25
 **"Trimite o noua cerere"** si "Toate cererile

**Comment by JohnPaul97** [comment]
error
Those text-inputs must be enabled.

**Comment by norberthcsorba** [comment]
@JohnPaul97 there's no need for those text fields (nume, prenume) to be enabled, we decided that. They will be automatically filled because the logged in Donor will have a Donor object stored which contains the firstname and lastname. The Donor won't change his name with time so he shouldn't be able to:
1. Have an account as Darius Galanton
2.Complete form with name Norberth Csorba
3.Complete another form with name Paul Cus
Which one is his real name, after all???

So it's not a mistake. It's what we asked for :)

**Pull request merged**